packages/aws-rfdk/lib/deadline/lib/repository.ts

```diff
@@ -526,6 +579,23 @@ export class Repository extends Construct implements IRepository {
 
     this.version = props.version;
 
+    this.secretsManagementSettings = {
+      enabled: props.secretsManagementSettings?.enabled ?? true,
+      credentials: props.secretsManagementSettings?.credentials ??
+        ((props.secretsManagementSettings?.enabled ?? true) ? new Secret(this, 'SMAdminUser', {
+          description: 'Admin credentials for Secret Management',
```

Comment:
```suggestion
          description: 'Admin credentials for Deadline Secrets Management',
```
We should be clear this is for Deadline. I could see it being confusing otherwise.